tools/dfx-orbit/src/lib.rs

```diff
@@ -150,9 +150,18 @@ impl DfxOrbit {
             .read_state_canister_info(canister_id, "controllers")
             .await?;
         let value: ciborium::Value = ciborium::from_reader(&blob[..])?;
-        let ciborium::Value::Array(array) = value else {
-            bail!("Expected an array as result from controllers endpoint")
+
+        let array = match value {
+            ciborium::Value::Array(array) => array,
+            ciborium::Value::Tag(_, inner) => {
+                let Ok(array) = inner.into_array() else {
+                    bail!("Expected an array as result from controllers endpoint");
+                };
+                array
+            }
```

Comment:
Why was this not needed before?

Reply:
There was a breaking change in the station, and we could not use this function. Therefore I did not surface the call to the  CLI level, and also I didn't have time to implement a test before.

I did both of these things now, and found out that there was a bug in the cbor parsing, while testing. So this was needed all along.